JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,19 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  if (seconds > 0) {
+    let secondsT = seconds % 60;
+    let hoursT = Math.floor((seconds - secondsT) / 60 / 60);
```

Comment:
`60` is being used multiple times, if this is a conversion factor I'd recommend to declare as const so you can reuse it instead of hardcoding the value, the same recommendation could apply for `seconds - secondsT`